src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+**Relay**
```

Comment:
This needs to be alphabetical in the list.

Reply:
sur, will make change